utm.yaml

```diff
@@ -1317,6 +1271,213 @@ components:
         status:
           $ref: '#/components/schemas/UssAvailabilityStatus'
 
+    RequestResponse:
+      type: object
+      description: Details of a request/response data exchange.
+      required:
+      - url
+      - method
+      - request_time
+      - request_body
+      - response_time
+      - response_body
+      - response_code
```

Comment:
Not every request receives a response, so we should make sure we can express those conditions.  I recommend making this field optional with a default value of 0, and 0 indicates that a response was not received.